doc/examples/jupyter_notebook/jupyter.json

```diff
@@ -4,7 +4,7 @@
     "external_port": 30888
   },
   "transform": {
-    "image": "pachyderm_jupyter",
+    "image": "dwhitena/pachyderm_jupyter",
```

Comment:
Why does this need to point to the copy in your docker account? Generally I'd prefer people be able to build it and use the local docker image. If for some reason we think they need an external image reference (e.g. we think building the docker image is too hard), then it should be under the pachyderm account.